src/cltkv1/morphology/Morphosyntax.py

```diff
@@ -0,0 +1,330 @@
+"""A module for representing universal morphosyntactic feature bundles."""

```

Comment:
@clemsciences Please look at this module and write to us, about whether it could work for your work for Old Norse. If not, what else would be necessary?

Reply:
Ok, I'm going to look at it.

Reply:
Thank you Clement! I might merge this soon (within a day or two), however you are welcome to share your feedback for us anytime.

Reply:
This module is more general than [this part of inflection module](https://github.com/cltk/cltk/blob/master/cltk/inflection/utils.py). The inflection module I implemented can be based on `morphosyntax.py`. I will see how exactly later.